source/about/mattermost-v10-changelog.md

```diff
@@ -102,6 +102,7 @@ New setting options were added to ``config.json``. Below is a list of the additi
  - Updated minimum Safari version to 17.4+ and minimum Firefox version to 119+.
 
 ### Important Upgrade Notes
+ - Both Mattermost v10.5.0 and v10.5.1 come bundled with a Jira plugin version that contains a bug causing the plugin configuration to disappear. We strongly discourage upgrading to these versions to avoid this issue. If you have already upgraded to 10.5.0 or 10.5.1, we recommend downgrading the Jira plugin to a stable version.
```

Comment:
```suggestion
 - Both Mattermost v10.5.0 and v10.5.1 come bundled with a Jira plugin version that contains a bug causing the plugin configuration to disappear. We strongly discourage upgrading to these versions to avoid this issue. If you have already upgraded to 10.5.0 or 10.5.1, we recommend upgrading the Jira version to 4.2.1 or ideally to upgrade both Mattermost and Jira to the latest version.
```

Something like this would be better, additionally 10.5.2 also contains security fixes so we should encourage everyone to upgrade to that release.